libcudacxx/test/libcudacxx/std/utilities/tuple/tuple.general/ignore.pass.cpp

```diff
@@ -10,45 +10,69 @@
 
 // constexpr unspecified ignore;
 
-// UNSUPPORTED: c++98, c++03
-
 #include <cuda/std/cassert>
 #include <cuda/std/tuple>
 
 #include "test_macros.h"
 
+static_assert(cuda::std::is_trivial<decltype(cuda::std::ignore)>::value, "");
+
+#if TEST_STD_VER >= 2017
+[[nodiscard]] __host__ __device__ constexpr int test_nodiscard()
+{
+  return 8294;
+}
+#endif
+
 __host__ __device__ TEST_CONSTEXPR_CXX14 bool test()
 {
-  { // Test that std::ignore provides constexpr converting assignment.
+  {
+    auto& ignore_v = cuda::std::ignore;
+    unused(ignore_v);
+  }
+
+  { // Test that std::ignore provides converting assignment.
     auto& res = (cuda::std::ignore = 42);
+    static_assert(noexcept(res = (cuda::std::ignore = 42)), "");
     assert(&res == &cuda::std::ignore);
   }
-  { // Test that cuda::std::ignore provides constexpr copy/move constructors
+  { // Test bit-field binding.
+    struct S
+    {
+      unsigned int bf : 3;
+    };
+    S s{0b010};
```

Comment:
Important: Binary literals require C++14, please use an ordinary integer literal to keep the test compiling with C++11.